prime-router/src/test/kotlin/SubmissionReceiverTests.kt

```diff
@@ -662,6 +817,19 @@ class SubmissionReceiverTests {
 
     @Test
     fun `test ELR receiver validateAndMoveToProcessing, inactive sender`() {
+        callELRReceiverValidateAndMoveToProcessing(Report.Format.HL7, hl7_record)
+    }
+    @Test
+    fun `test ELR receiver validateAndMoveToProcessing, HL7_BATCH format with header`() {
+        callELRReceiverValidateAndMoveToProcessing(Report.Format.HL7_BATCH, hl7_record_batch_headers)
+    }
+
+    @Test
+    fun `test ELR receiver validateAndMoveToProcessing, HL7_BATCH format no header, multiple records`() {
+        callELRReceiverValidateAndMoveToProcessing(Report.Format.HL7_BATCH, hl7_multiple_records_no_headers)
+    }
+
+    private fun callELRReceiverValidateAndMoveToProcessing(format: Report.Format, content: String) {
```

Comment:
(optional) It wasn't immediately obvious to me from the name here that this was a helper function for testing. Looking at the code, and seeing where the function lives makes it clear, so it might not need a change, but I wonder if `testELRReceiverValidateAndMoveToProcessing` might be better to use? Totally up to you!